Packs/HashiCorp-Vault/Integrations/HashiCorpVault/README.md

```diff
@@ -1,1133 +1,421 @@
-                {
-                    "Capabilities": [
-                        "update"
-                    ],
-                    "Path": "sys/wrapping/wrap"
-                },
-                {
-                    "Capabilities": [
-                        "update"
-                    ],
-                    "Path": "sys/tools/random"
-                },
-                {
-                    "Capabilities": [
-                        "read"
-                    ],
-                    "Path": "sys/mounts"
-                },
-                {
-                    "Capabilities": [
-                        "update"
-                    ],
-                    "Path": "sys/wrapping/unwrap"
-                },
-                {
-                    "Capabilities": [
-                        "read",
-                        "list"
-                    ],
-                    "Path": "sys/policy"
-                },
-                {
-                    "Capabilities": [
-                        "read",
-                        "list"
-                    ],
-                    "Path": "sys/policy/*"
-                },
-                {
-                    "Capabilities": [
-                        "read",
-                        "list"
-                    ],
-                    "Path": "sys/policies/*"
-                },
-                {
-                    "Capabilities": [
-                        "read",
-                        "delete"
-                    ],
-                    "Path": "secret/*"
-                }
-            ]
-        }
-    }
-}
-</pre>
-<h5>Human Readable Output</h5>
-<p><a href="https://user-images.githubusercontent.com/35098543/50423429-c3b65d00-085d-11e9-88b3-6a66ac38ee3c.png" target="_blank" rel="noopener noreferrer"><img src="https://user-images.githubusercontent.com/35098543/50423429-c3b65d00-085d-11e9-88b3-6a66ac38ee3c.png" alt="image"></a></p>
-<h3 id="h_97508506437331545850383612">11. Seal a vault</h3>
-<hr>
-<p>If you suspect your data has been compromised, you can seal your vault to prevent access to your secrets.</p>
-<h5>Base Command</h5>
-<p><code>hashicorp-seal-vault</code></p>
-<h5>Input</h5>
-<p>There are no input arguments for this command.</p>
-<h5>Context Output</h5>
-<p>There is no context output for this command.</p>
-<h5>Command Example</h5>
-<pre>!hashicorp-seal-vault
-</pre>
-<h5>Human Readable Output</h5>
-<p><a href="https://user-images.githubusercontent.com/35098543/50424561-c53c5100-086e-11e9-90f9-54795257b202.png" target="_blank" rel="noopener noreferrer"><img src="https://user-images.githubusercontent.com/35098543/50424561-c53c5100-086e-11e9-90f9-54795257b202.png" alt="image"></a></p>
-<h3 id="h_24467581039191545850479673">12. Unseal a vault</h3>
-<hr>
-<p>Use a single master key share to unseal the vault. If the master key shares threshold is met, vault will attempt to unseal the vault. Otherwise, this API must be called until the threshold is met.</p>
```

Comment:
Is this correct....
the **vault** will attempt to unseal the vault?
Should it be 
the **key** will attempt to unseal the vault?